### PR TITLE
Quest page changes

### PR DESCRIPTION
**Pull request by crmsnwl**
Minor fixes according to comments

### CODE REVIEW

File: CityPuzzle/Main Game/QuestPage.xaml.cs [hunk 1]
```diff
@@ -19,7 +19,6 @@ public partial class QuestPage : ContentPage
         private double QuestLng;
         private List<Puzzle> Target;
         public static Puzzle QuestInProgress;
-        public const Double HumanSpeed = 2.23;
```
[the remark this answers is not in the document]
Comment (crmsnwl) [reply CityPuzzle/Main Game/QuestPage.xaml.cs:22]: Masks primary function is not to provide information on distance left but to reveal how the location looks like. So radars function ahould be giving info on direction and distance

[the remark this answers is not in the document]
Comment (Justuxs) [reply CityPuzzle/Main Game/QuestPage.xaml.cs:22]: Okay, it's fine.

File: CityPuzzle/Main Game/QuestPage.xaml.cs [hunk 2]
```diff
@@ -224,30 +223,44 @@ public int CountPages()
...
-                Console.WriteLine("Updatinu radara - i " + state.ToString() + ".gif"+" speed "+ speed+" "+ direction);
-                distCheck = distChange;
-                nowSize = CountPages();
+                distChange = DistanceToPoint(QuestLat, QuestLng);
+
+                if (distChange != distCheck)
+                {
+                    if (distChange < distCheck && distChange <= (startDist / 2))
+                    {
+                        myRadar = (Radar)4;
```
Comment (AllBill127) [inline CityPuzzle/Main Game/QuestPage.xaml.cs:245]: Use enums instead of casts

File: CityPuzzle/Main Game/QuestPage.xaml.cs [hunk 3]
```diff
@@ -224,30 +223,44 @@ public int CountPages()
             int stackSize = existingPages.Count;
             return stackSize;
         }
-        async void RadarFoo(){
+        async void RadarFoo()
```
Comment (AllBill127) [inline CityPuzzle/Main Game/QuestPage.xaml.cs:226]: Give proper name to method

**Comment by AllBill127** [comment]
Thank you for the changes. The code looks good. If no more changes will be made here you can merge into main

**Pull request merged**